src/Microsoft.Azure.WebJobs.Extensions.Storage/Blobs/Bindings/WatchableCloudBlobStream.cs

```diff
@@ -153,5 +187,40 @@ public ParameterLog GetStatus()
                 return null;
             }
         }
+
+        private void Log()
+        {
+            if (_logged)
+            {
```

Comment:
Is this needed? As the `Log()` is only called on `close()` ?

Reply:
Discussed with @pragnagopa - I noticed that `Dispose` was being called twice so put this check here to avoid duplicate logging.